caffe2/python/onnx/frontend.py

```diff
@@ -504,14 +539,28 @@ def _annotate_consumed(cls, graph_def):
                     consumes,
                 )])
 
+    @classmethod
+    def _filter_fake_init(cls, init_net, value_info):
+        if init_net:
+            index = 0
+            while index < len(init_net.op):
```

Comment:
Python itself actually has a `filter` function. You can try using that to save some lines.

Reply:
Here, we cannot have assignment like `init_net.op[i] = new_op`. But applied something similar to make the code look nicer.